### PR TITLE
chore revscript: conversion talkactions

### PR DESCRIPTION
**Pull request by dudantas**

### CODE REVIEW

File: data/scripts/talkactions/god/attributes.lua [hunk 1]
```diff
@@ -1,10 +1,9 @@
+local talk = TalkAction("/attr")
+
 local itemFunctions = {
 	["actionid"] = { isActive = true, targetFunction = function (item, target) return item:setActionId(target) end },
 	["action"] = { isActive = true, targetFunction = function (item, target) return item:setActionId(target) end },
 	["aid"] = { isActive = true, targetFunction = function (item, target) return item:setActionId(target) end },
-	["uniqueid"] = { isActive = true, targetFunction = function (item, target) return item:setAttribute(ITEM_ATTRIBUTE_UNIQUEID, target) end },
```
Comment (ghost) [inline data/scripts/talkactions/god/attributes.lua]: pq removeu essa parte

File: data/scripts/talkactions/god/create summon.lua [hunk 2]
```diff
@@ -1,15 +1,18 @@
-function onSay(player, words, param)
-	if not player:getGroup():getAccess() then
+local talk = TalkAction("/s")
```
Comment (ghost) [inline data/scripts/talkactions/god/create summon.lua:1]: não é /summon ?

**Review by ghost — CHANGES_REQUESTED** [review]

File: data/scripts/talkactions/god/create summon.lua (hunk 2 above)
Comment (dudantas) [reply data/scripts/talkactions/god/create summon.lua:1]: no

File: data/scripts/talkactions/god/add skill.lua [hunk 3]
```diff
@@ -1,3 +1,6 @@
+talk = TalkAction("/addskill")
```
Comment (gpedro) [inline data/scripts/talkactions/god/add skill.lua]: não precisa definir ela como `local`?

File: data/scripts/talkactions/god/add skill.lua [hunk 4]
```diff
@@ -15,29 +18,30 @@ local function getSkillId(skillName)
...
+-- function for the talkaction /addskill 
 local function getExpForLevel(level)
 	level = level - 1
 	return ((50 * level * level * level) - (150 * level * level) + (400 * level)) / 3
 end
 
-function onSay(player, words, param)
-	if not player:getGroup():getAccess() then
+function talk.onSay(player, words, param)
+	if(not player:getGroup():getAccess()) or player:getAccountType() < ACCOUNT_TYPE_GOD then
```
Comment (gpedro) [inline data/scripts/talkactions/god/add skill.lua]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/add skill.lua [hunk 5]
```diff
@@ -15,29 +18,30 @@ local function getSkillId(skillName)
...
 		return true
 	end
 
-	if player:getAccountType() < ACCOUNT_TYPE_GOD then
+	if(param == "") then
```
Comment (gpedro) [inline data/scripts/talkactions/god/add skill.lua]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/animationeffect.lua [hunk 6]
```diff
@@ -0,0 +1,39 @@
+local talk = Talkaction("/effect")
+
+function onSay(player, words, param)
+	if(not player:getGroup():getAccess()) or player:getAccountType() < ACCOUNT_TYPE_GOD then
```
Comment (gpedro) [inline data/scripts/talkactions/god/animationeffect.lua]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/animationeffect.lua [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+		return true
+	end
+
+	if(param == "") then
```
Comment (gpedro) [inline data/scripts/talkactions/god/animationeffect.lua]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/attributes.lua [hunk 8]
```diff
@@ -48,13 +50,13 @@ local playerFunctions = {
 	["premium"] = { isActive = true, targetFunction = function (player, target) return player:addPremiumDays(target) end }
 }
 
-function onSay(player, words, param)
+function talk.onSay(player, words, param)
 	if(not player:getGroup():getAccess()) or player:getAccountType() < ACCOUNT_TYPE_GOD then
 		return true
 	end
 
 	if(param == "") then
```
Comment (gpedro) [inline data/scripts/talkactions/god/attributes.lua]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/attributes.lua [hunk 9]
```diff
@@ -48,13 +50,13 @@ local playerFunctions = {
 	["premium"] = { isActive = true, targetFunction = function (player, target) return player:addPremiumDays(target) end }
 }
 
-function onSay(player, words, param)
+function talk.onSay(player, words, param)
 	if(not player:getGroup():getAccess()) or player:getAccountType() < ACCOUNT_TYPE_GOD then
```
Comment (gpedro) [inline data/scripts/talkactions/god/attributes.lua]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/attributes.lua [hunk 10]
```diff
@@ -66,38 +68,41 @@ function onSay(player, words, param)
 	if(itemFunction and itemFunction.isActive) then
 		local item = Tile(position):getTopVisibleThing(player)
 		if(not item or not item:isItem()) then
```
Comment (gpedro) [inline data/scripts/talkactions/god/attributes.lua:70]: this parenthesis is not necessary.

File: data/scripts/talkactions/god/attributes.lua [hunk 11]
```diff
@@ -66,38 +68,41 @@ function onSay(player, words, param)
...
-			player:sendTextMessage(MESSAGE_STATUS_CONSOLE_BLUE, "Item not found.")
+			player:sendCancelMessage("Item not found.")
 			return false
 		end
 		if(itemFunction.targetFunction(item, split[2])) then
```
Comment (gpedro) [inline data/scripts/talkactions/god/attributes.lua:74]: this parenthesis is not necessary.

**Comment by gpedro** [comment]
just check lua code style .... except this, lgtm

**Pull request merged**